test/controllers/check.your.answers.controller.spec.ts

```diff
@@ -489,8 +488,7 @@ describe("POST tests", () => {
   });
 
   test(`redirect the ${CONFIRMATION_PAGE} page after a successful post from ${CHECK_YOUR_ANSWERS_PAGE} page`, async () => {
-    mockIsActiveFeature.mockReturnValueOnce( false ); // For Refresh token
-    mockIsActiveFeature.mockReturnValueOnce( false ); // For Save and Resume
+    mockIsActiveFeature.mockReturnValueOnce( false );
```

Comment:
Minor, but just wonder if it's worth leaving the comment in on this line and one below, stating what feature flag the 'false' value refers to? I'm guessing 'save and resume' now.